content/webapp/pages/stories.tsx

```diff
@@ -81,8 +84,12 @@ const pageDescription =
 export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
   async context => {
     const serverData = await getServerData(context);
-    const { page = 1, memoizedPrismic } = context.query;
```

Comment:
I removed this here because `/stories` never links to `?page=2`, `?page=3`, and so on – it links out to `/articles`, which is a different page.

The reason I didn't migrate it straight across is because the type checker doesn't like the `page` parameter. Its type is actually `string | string[] | number | undefined`, depending on what the user passes as a query parameter. You can see this go wrong if you request e.g. https://wellcomecollection.org/articles?page=2&page=3 – when `page = ['2', '3']` and the Prismic query explodes.

So I think what we want is a method that extracts the `page` from the query parameters, which is guaranteed to return a `number | undefined`, and absorbs weirdness like strings or multiple strings – but that should be a separate patch.

Reply:
I suppose the question here is whether we care about people who have existing links to paginated stories, but I think we probably don't as the pages aren't stable anyway

Reply:
I'm not sure anybody should have those links anyway, so I definitely don't care.